### PR TITLE
Update README.md put cleanup seperately

### PR DESCRIPTION
**Pull request by cccsss01**

### CODE REVIEW

File: charts/spire/README.md [hunk 1]
```diff
@@ -68,6 +68,14 @@ helm upgrade --install -n spire-mgmt spire spire --repo https://spiffe.github.io
 
 We only support upgrading one major version at a time. Version skipping isn't supported.
 
+### Clean up
```
Comment (kfox1111) [inline charts/spire/README.md:71]: Didn't notice before, but the cleanup section is nested here inside the middle of the upgrade notes section. It needs to be moved out to its own section.

Comment (cccsss01) [reply charts/spire/README.md:71]: Isn't this moved out with the 3 hashes?

File: charts/spire/README.md [hunk 2]
```diff
@@ -64,6 +64,14 @@ helm upgrade --install -n spire-mgmt spire-crds spire-crds --repo https://spiffe
 helm upgrade --install -n spire-mgmt spire spire --repo https://spiffe.github.io/helm-charts-hardened/ -f your-values.yaml
 ```
 
+### Clean up
```
Comment (kfox1111) [inline charts/spire/README.md:67]:
```suggestion
## Clean up
```

Comment (kfox1111) [reply charts/spire/README.md:67]: Back to 2 so it is its own section at the same level as 'Install Instructions' and 'Upgrade notes' and then I think its good to go.

**Review by kfox1111 — APPROVED** [review]
LGTM. Thanks! :)

**Pull request merged**